webapp/jaasai/views.py

```diff
@@ -126,6 +135,56 @@ def blog_feed():
     return jsonify(response)
 
 
+@jaasai.route("/robots.txt")
+def static_from_root():
+    return send_from_directory(current_app.static_folder, request.path[1:])
+
+
+@jaasai.route("/sitemap.xml")
+def sitemap():
+    return render_template("sitemaps/sitemap.xml")
+
+
+@jaasai.route("/sitemap-base.xml")
+def sitemap_base():
+    return render_template(
+        "sitemaps/sitemap-base.xml",
+        context={
+            "pages": [
```

Comment:
Is there no way to generate this? This strikes me as a place that will quickly get out of date.

Reply:
There are, by they all involve a lot more complex code and will still require us to whitelist/blacklist from the generate list. Not sure which is better...